docs/configuration/actions.md

```diff
@@ -31,6 +31,31 @@ Actions have a default behaviour which you can override, hooks don't have any -
 hooks are a way for you to perform operations following a certain packit event,
 e.g. cloning an upstream repo.
 
+
+:::caution
+
+As other keys, the `actions` can be defined on the top, package or job level.
+Just be aware that when overriding, the whole `action` mapping is replaced
+instead of merging.
```

Comment:
```suggestion
Like other keys, the `actions` can be defined on the top, package or job level.
Be aware that when overriding, the whole `action` mapping is replaced
instead of merging.
```